test/TestProjects/TypeSchemaMapping/Generated/MainRestClient.cs

```diff
@@ -57,53 +57,29 @@ internal HttpMessage CreateOperationRequest(CustomizedModel body)
 
         /// <param name="body"> The Model to use. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public async ValueTask<Response<CustomizedModel>> OperationAsync(CustomizedModel body = null, CancellationToken cancellationToken = default)
+        public async ValueTask<Response> OperationAsync(CustomizedModel body = null, CancellationToken cancellationToken = default)
         {
             using var message = CreateOperationRequest(body);
             await _pipeline.SendAsync(message, cancellationToken).ConfigureAwait(false);
             switch (message.Response.Status)
             {
                 case 200:
-                    {
-                        CustomizedModel value = default;
-                        using var document = await JsonDocument.ParseAsync(message.Response.ContentStream, default, cancellationToken).ConfigureAwait(false);
-                        if (document.RootElement.ValueKind == JsonValueKind.Null)
-                        {
-                            value = null;
-                        }
-                        else
-                        {
-                            value = CustomizedModel.DeserializeCustomizedModel(document.RootElement);
-                        }
-                        return Response.FromValue(value, message.Response);
-                    }
```

Comment:
The response type changed. Is it because it is an LRO now?

Reply:
Yes, I turned it into an LRO to exercise in a test.